assets/js/modules/analytics-4/datastore/base.js

```diff
@@ -27,6 +27,7 @@ const baseModuleStore = Modules.createModuleStore( 'analytics-4', {
 	settingSlugs: [
 		// TODO: This can be uncommented when Analytics and Analytics 4 modules are officially separated.
 		// 'accountID',
+		'adsConversionID',
```

Comment:
I think it would be better to also add this here only as a commented-out setting, the same way as `accountID`. As decided while discussing #3165, we shouldn't double-store or sync values between both modules. By adding this here as commented out we indicate that in principle it needs to be stored in the module as well, but let's not worry about it for now, since `analytics-4` is only available with `analytics` active anyway.

Reply:
Sorry, looks like I missed that issue. You mentioned that we shouldn't double-store or sync (I think you also mean double-storing here?) but what about proxying the value? Not for all the settings but only those that are relevant from the other module. That would allow us to use settings in GA4 from UA settings without requiring any special handling from the consumer. We can encapsulate this handling in its settings class entirely so we only have one place to change later when they are decoupled. I have an example in here in one commit, so we can revert if you'd prefer.